examples/computecloth/computecloth.cpp

```diff
@@ -382,11 +417,32 @@ class VulkanExample : public VulkanExampleBase
 		VkCommandBuffer copyCmd = vulkanDevice->createCommandBuffer(VK_COMMAND_BUFFER_LEVEL_PRIMARY, true);
 		VkBufferCopy copyRegion = {};
 		copyRegion.size = storageBufferSize;
+		vkCmdCopyBuffer(copyCmd, stagingBuffer.buffer, storageBuffers.input.buffer, 1, &copyRegion);
```

Comment:
So this dual copy is a little weird.  

Basically we want the input buffer to contain the data at the beginning of the compute stage, so we populate it here, but we also want the output buffer to be owned by the graphics queue initially, so we also populate it here. 
Technically we don't really need to populate it, we just need to have it accessed by the graphics queue in some way to take initial ownership, so the initial release barrier pairs up.